packages/@coorpacademy-progression-engine/src/compute-next-step/compute-next-step.js

```diff
@@ -357,4 +357,22 @@ const computeNextStep = (
   return null;
 };
 
-export default computeNextStep;
+export const computeNextStepForReview = (
+  config: Config,
+  _state: State | null,
+  availableContent: AvailableContent
+): Result => {
+  const nextSlide = get(['0', 'slides', '0'], availableContent);
+  if (!nextSlide) {
+    return null;
+  }
+
+  return {
+    nextContent: {
+      type: 'slide',
+      ref: nextSlide._id
+    },
+    instructions: null,
+    isCorrect: false
```

Comment:
pourquoi false?

Reply:
@esa-coorp c'est un code itératif, la vrai valeur sera pris du state mais pour la création le state est null. La logique pour le isCorrect viendra dans le ticket de computeNextContentAfterAnswer